packages/ui/src/memberships/components/MemberListItem/Member.tsx

```diff
@@ -18,14 +17,7 @@ export const MemberListItem = ({ member }: { member: Member }) => {
   const council = useObservable(api?.query.council.councilMembers(), [connectionState])
   const councilMembersIds = council?.map(({ membership_id }) => membership_id.toNumber()) ?? []
   const isCouncil = (id: number) => councilMembersIds.includes(id)
-  const { showModal } = useModal()
-  const showMemberModal = useCallback(
-    (event?: React.MouseEvent<HTMLElement>) => {
-      event?.stopPropagation()
-      member && showModal<MemberModalCall>({ modal: 'Member', data: { id: member.id } })
```

Comment:
@ClumsyVlad Do you remember why this was needed?

Reply:
There was a conflict caused by nesting of interactive elements, but now I think this is outdated part. Could anyone test this element to avoid bugs with interaction with inner elements?